boards/arm/nrf5340pdk_nrf5340/nrf5340_cpunet_reset.c

```diff
@@ -51,12 +58,13 @@ static int remoteproc_mgr_boot(struct device *dev)
 	remoteproc_mgr_config();
 #endif /* !CONFIG_TRUSTED_EXECUTION_NONSECURE */
 
-#if (DT_IPC_SHM_BASE_ADDRESS != 0)
+#if (SHM_BASE_ADDRESS != 0)
```

Comment:
are you fine w/the assumption that zephyr,ipc-shm is always set?  If not seems like you'll have a case that SHM_BASE_ADDRESS will not be defined if zephyr,ipc-shm doesn't exist.

Reply:
zephyr,ipc-shm should always be set, but let's make it more solid and add and #idfef here?

Reply:
updated it now @galak